tests/system/action/motion_category/test_create.py

```diff
@@ -0,0 +1,43 @@
+from tests.system.base import BaseSystemTestCase
+from tests.util import get_fqid
+
+
+class MotionCategorySystemTest(BaseSystemTestCase):
+    def test_create(self) -> None:
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion_category.create",
+                    "data": [{"name": "test_Xcdfgee"}],
+                }
+            ],
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assert_model_exists(get_fqid("motion_category/1"))
+        model = self.datastore.get(get_fqid("motion_category/1"))
+        assert model.get("name") == "test_Xcdfgee"
+
+    def test_create_empty_data(self) -> None:
+        response = self.client.post(
+            "/", json=[{"action": "motion_category.create", "data": [{}]}],
+        )
+        self.assertEqual(response.status_code, 400)
+        self.assertIn(
+            "data[0] must contain [\\'name\\'] properties", str(response.data),
+        )
+
+    def test_create_wrong_field(self) -> None:
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion_category.create",
+                    "data": [{"wrong_field": "text_AefohteiF8"}],
+                }
+            ],
+        )
+        self.assertEqual(response.status_code, 400)
+        self.assertIn(
+            "data[0] must contain [\\'name\\'] properties", str(response.data),
+        )
```

Comment:
After adjusting the required properties, add some more tests here, e.g. providing no `meeting_id` or a non-existent one.

Reply:
Do we really want that kind of tests here? I don't think so, one wrong field test is enough.

Reply:
At least a non-existing meeting id should be tested, because we have to make sure that we don't create models for non-existing meetings.

Reply:
\*ping\*. Also, please add a test with all possible fields

Reply:
Test all fields in the create/update tests.
Add a test on not existing meeting.